sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/CompletionsUsagePromptTokensDetails.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * The CompletionsUsagePromptTokensDetails model.
+ */
+@Immutable
+public final class CompletionsUsagePromptTokensDetails
+    implements JsonSerializable<CompletionsUsagePromptTokensDetails> {
+
+    /*
+     * The number of cached prompt tokens.
+     */
+    @Generated
+    private Integer cachedTokens;
+
+    /**
+     * Creates an instance of CompletionsUsagePromptTokensDetails class.
+     */
+    @Generated
+    private CompletionsUsagePromptTokensDetails() {
+    }
+
+    /**
+     * Get the cachedTokens property: The number of cached prompt tokens.
+     *
+     * @return the cachedTokens value.
+     */
+    @Generated
+    public Integer getCachedTokens() {
```

Comment:
We should call this `getCachedTokenCount()`

Reply:
We use `total/completion/prompt` tokens for the token counts under CompletionUsage. Both new parameters, promp_tokens_details and completion_tokens_details are inside of CompletionUsage. We show keep it aligned with the same name pattern.

Reply:
Create an issue to address it in typespec: https://github.com/Azure/azure-sdk-for-java/issues/43120